syntax = "proto3";


service layer2_out{
	rpc request(DL_request) returns (DL_response){}
}

message DL_request{
	int32 state = 1;

}

message DL_response{
	bytes x_train = 1;	
	bytes y_train = 2;
}
